.github/workflows/test-pg_search.yml

```diff
@@ -152,7 +152,15 @@ jobs:
           RUST_BACKTRACE=1 cargo test --jobs $(nproc) --features pg${{ matrix.pg_version }} --no-default-features
 
       - name: Run pg_search Regression Tests Against distro-managed Postgres
-        run: cargo pgrx regress --package pg_search pg${{ matrix.pg_version }} --features=icu
+        run: |
+          output=$(cargo pgrx regress --package pg_search pg${{ matrix.pg_version }} --features=icu)
+          echo "$output"
+          if echo "$output" | grep -q "failed="; then
+            failed_count=$(echo "$output" | grep -o "failed=[0-9]*" | cut -d'=' -f2)
+            if [ "$failed_count" -gt 0 ]; then
+              exit 1
+            fi
+          fi
```

Comment:
When run with `--auto`, `regress` return 1 when it fails: could probably use that mode.